client/src/components/_common/UserSearchbar/UserSearchbar.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
we should convey to the user that we are performing the searching (i.e. https://jira.tacc.utexas.edu/browse/FP-120)

Reply:
@nathanfranklin Is that the right task to link?

Reply:
It is not :).   fixed above and in PR description 👍